wcfsetup/install/files/lib/action/MessageQuoteAction.class.php

```diff
@@ -131,12 +132,10 @@ public function execute()
         WCF::getSession()->disableUpdate();
 
         if ($returnValues !== null) {
-            // send JSON-encoded response
-            \header('Content-type: application/json; charset=UTF-8');
-            echo JSON::encode($returnValues);
+            return new JsonResponse($returnValues);
+        } else {
+            return new EmptyResponse(200);
```

Comment:
You can get rid of the `else` here.

Reply:
In this case it's intentional to make it more clear that these are the only 2 cases. An early return IMO would not be as fitting.

The `EmptyResponse` case can also be removed once the `markForRemoval` action is removed (I deprecated it in a previous PR).